Community-Supported/clouddb-extractor/README.md

```diff
@@ -0,0 +1,68 @@
+# clouddb-extractor
+## __clouddb-extractor__
+
+![Community Supported](https://img.shields.io/badge/Support%20Level-Community%20Supported-53bd92.svg)
+
+__Current Version__: 1.0
+
+Cloud Database Extractor Utility - This sample shows how to extract data from a cloud database to a published hyper extract and append/update/delete rows to keep up to date.
+
+# Overview
+This package defines a standard Extractor Interface which is extended by specific implementations
+to support specific cloud databases.  For most use cases you will probably only ever call the
+following methods:
+* __load_sample__ - Loads a sample of rows from source_table to Tableau Server
```

Comment:
I misunderstood `load_sample` at first. So far, I thought this would be "load some hard-coded sample data". What it actually means is "load a subset of the data".
A name like "load_preview" would have been more intuitive to me

Reply:
I'm using the term "Sample" here in the same sense as which it is used in Tableau Prep.  So let's keep as is to remain consistent with the rest of the toolset?